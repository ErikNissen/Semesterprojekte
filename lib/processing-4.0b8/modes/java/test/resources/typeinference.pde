version https://git-lfs.github.com/spec/v1
oid sha256:65c996751f67ab3216cb1984f8557878d92e4ed8b88537146086d54dcada618b
size 366
