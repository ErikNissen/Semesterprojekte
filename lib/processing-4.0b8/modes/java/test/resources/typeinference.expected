version https://git-lfs.github.com/spec/v1
oid sha256:44eb0b14ebfed8eaf644fbe8afb7008d6582096cd7e63aad94bc62fb98005ac8
size 1021
